src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-discovery-modal/iscsi-target-discovery-modal.component.html

```diff
@@ -19,8 +19,9 @@
             <input id="user"
                    class="form-control"
                    formControlName="user"
-                   type="text">
-            <span class="help-block"
+                   type="text"
+                   autocomplete="off">
+            <span class="invalid-feedback"
```

Comment:
`invalid-feedback` is not available in nautilus
```suggestion
            <span class="help-block"
```

Reply:
Thanks, updated!